draft-ietf-scitt-architecture.md

```diff
@@ -394,7 +392,7 @@ In COSE, an Envelope consists of a protected header (included in the Issuer's si
 
 Equivocation:
 
-: a state where it is possible for a Transparency Service to provide different views of its Append-only log to Relying Parties about the same Artifact {{EQUIVOCATION}}.
+: a state where it is possible for a Transparency Service to provide different Verifiable Data Structures to Relying Parties about the same Artifact {{EQUIVOCATION}}.
```

Comment:
```suggestion
: a state where a Transparency Service provides incompatible Verifiable Data Proofs to Relying Parties, containing conflicting claims about the Signed Statement bound at a given position in the Verifiable Data Structure {{EQUIVOCATION}}.
```

The service wouldn't provide the whole VDS in general, only VDPs/proofs extracted from it, so I think the practical manifestation of equivocation for observers is the presence of incompatible VPDs (which are certainly evidence of incompatible VDSs existing somewhere, but which the service will not usually provide wholesale).

Reply:
```suggestion
: a state where a Transparency Service provides inconsistent proofs to Relying Parties, containing conflicting claims about the Signed Statement bound at a given position in the Verifiable Data Structure {{EQUIVOCATION}}.
```
Maybe avoid introducing Verifiable Data Proofs ? I think we can reasonably just say "proofs" if the context is a Verifiable Data Structure

Also a nit: I think inconsistent is more accurate than "incompatible" especially given the other contexts interoperability and compatibility are used